File: examples/src/Text/SaveWithDefaultRegularFont.cs
using System
import aspose.pydrawing as drawing
using System.Drawing.Imaging
using System.IO
import aspose.slides as slides
using Aspose.slides.Export

namespace Aspose.slides.Examples.CSharp.text
{
    public class SaveWithDefaaultRegularFont
    {
        #/ <summary>
        #/ The code below demonstrates saving presentation to Html and Pdf with different default regular font.
        #/ </summary>
        public static void Run()
        {
            dataDir = RunExamples.GetDataDir_Text()
            outPath = RunExamples.OutPath

            using (Presentation pres = new Presentation(dataDir + "DefaultFonts.pptx"))
            {
                HtmlOptions htmlOpts = new HtmlOptions()
                htmlOpts.DefaultRegularFont = "Arial Black"
                pres.save(outPath + "Presentation-out-ArialBlack.html", SaveFormat.Html, htmlOpts)
                htmlOpts.DefaultRegularFont = "Lucida Console"
                pres.save(outPath + "Presentation-out-LucidaConsole.html", SaveFormat.Html, htmlOpts)

                PdfOptions pdfOpts = new PdfOptions()
                pdfOpts.DefaultRegularFont = "Arial Black"
                pres.save(outPath + "Presentation-out-ArialBlack.pdf", SaveFormat.Pdf, pdfOpts)
            }
        }
    }
}

File: examples/src/Text/SaveWithDefaultRegularFont.py
import aspose.pydrawing as drawing
import aspose.slides as slides

dataDir = "./examples/data/"
outDir = "./examples/out/"

with slides.Presentation(dataDir + "text_default_fonts.pptx") as pres:
    htmlOpts = slides.export.HtmlOptions()
    htmlOpts.default_regular_font = "Arial Black"
    pres.save(outDir + "text_Presentation-out-ArialBlack.html", slides.export.SaveFormat.HTML, htmlOpts)
    
    htmlOpts.default_regular_font = "Lucida Console"
    pres.save(outDir + "text_Presentation-out-LucidaConsole.html", slides.export.SaveFormat.HTML, htmlOpts)

    pdfOpts = slides.export.PdfOptions()
    pdfOpts.default_regular_font = "Arial Black"
    pres.save(outDir + "text_Presentation-out-ArialBlack.pdf", slides.export.SaveFormat.PDF, pdfOpts)